qiskit_experiments/quantum_volume/qv_experiment.py

```diff
@@ -0,0 +1,129 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2021.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+"""
+Quantum Volume Experiment class.
+"""
+
+from typing import Union, Iterable, Optional, List
+from numpy.random import Generator, default_rng
+from qiskit.providers.backend import Backend
+from qiskit.providers.options import Options
+
+try:
+    from qiskit import Aer
+
+    HAS_SIMULATION_BACKEND = True
+except ImportError:
+    HAS_SIMULATION_BACKEND = False
+
+from qiskit import QuantumCircuit
+from qiskit.circuit.library import QuantumVolume as QuantumVolumeCircuit
+from qiskit import execute
+from qiskit_experiments.base_experiment import BaseExperiment
+from .qv_analysis import QuantumVolumeAnalysis
+
+
+class QuantumVolume(BaseExperiment):
+    """Quantum Volume Experiment class
+
+    Experiment Options:
```

Comment:
Experiment doc string needs expanding, but this can be follow up PR.

Reply:
I will handle it in a seperate PR after this code is merged